pkg/internal/cluster/create/nodes.go

```diff
@@ -165,7 +165,7 @@ func nodesToCreate(cfg *config.Cluster, clusterName string) []nodeSpec {
 	isHA := controlPlanes > 1
 	// obtain IP family
 	ipv6 := false
-	if cfg.Networking.IPFamily == "ipv6" {
+	if cfg.Networking.IPFamily == "ipv6" || cfg.Networking.IPFamily == "dual-stack" {
```

Comment:
the family isn't ipv4ipv6?

Reply:
Indeed it did but was thinking that  dual-stack has better UX as a parameter, also is the way the feature is called in kubernetes
 I think ipv4ipv6 can be misleading, anyway, I'm totally open to any suggestion

Reply:
we should be consistent with upstream.

Reply:
The constants upstream are "IPv4" and "IPv6" for `service.Spec.IPFamily`. We don't carry `dualstack` anywhere as a configuration value/type value. So it should be `IPv4` or `IPv6` or `dualstack`. I don't think i have seen `ipv4ipv6` reference anywhere else. it is always `dualstack ipv4/ipv6` or just `dualstack`.

Reply:
thanks!

hmm, so this is the IP family "at the cluster scope", `dualstack` could work, no hyphens? hyphen? :+)

Reply:
API constant style would be `DualStack`.  We don't have an explict constant for that because we're not offering it for Services yet.

Reply:
this was moved to the full dualstack PR with the name `DualStack`, I will use this PR only for kindnet